_rules/SC3-1-2-lang.md

```diff
@@ -1,79 +1,82 @@
 ---
-rule_id: SC3-1-2-lang
-name: Languages within the body
-test_mode: automatic
-environment: DOM Structure
+name: Valid body lang attribute
```

Comment:
I suggest renaming the file to to SC3-1-2-lang-valid so that it matches the SC3-1-1-html-lang-valid in naming structure in relation to purpose.